frontend/src/modules/contributor/services/contributor.identities.api.service.ts

```diff
@@ -0,0 +1,46 @@
+import authAxios from '@/shared/axios/auth-axios';
+import { AuthService } from '@/modules/auth/services/auth.service';
+import { ContributorIdentity } from '@/modules/contributor/types/Contributor';
+
+export class ContributorIdentitiesApiService {
+  static async list(memberId: string, segments: string[]) {
+    const tenantId = AuthService.getTenantId();
+
+    return authAxios.get(
+      `/tenant/${tenantId}/member/${memberId}/identity`,
+      {
+        params: {
+          segments,
+        },
+      },
+    ).then(({ data }) => Promise.resolve(data));
+  }
```

Comment:
Should we also use this endpoint to get the list of identities in a profile? I thought we would also get data in parts